### PR TITLE
Genome filter

### PR DESCRIPTION
**Pull request by borismattijssen**
Jaa a visual genome filter!

### CODE REVIEW

**Comment by Delfil** [comment]
We should be consistent in constructors, or we give all classes without a constructor an empty public constructor, or we shouldn't allow empty public constructors. Also a javadoc saying "Constructor." isn't informative.

File: src/main/java/tudelft/ti2806/pl3/sidebar/genomeFilter/GenomeFilterView.java [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+import javax.swing.JScrollPane;
+import javax.swing.ListSelectionModel;
+
+/**
+ * View that shows the genome filter.
+ * Created by Boris Mattijssen on 12-05-15.
+ */
+public class GenomeFilterView extends JPanel implements SideBarViewInterface {
+
+	private JList list;
```
Comment (Delfil) [inline src/main/java/tudelft/ti2806/pl3/sidebar/genomeFilter/GenomeFilterView.java]: Raw type

File: src/main/java/tudelft/ti2806/pl3/sidebar/genomeFilter/GenomeFilterView.java [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+		return button;
+	}
+
+	/**
+	 * Creates the JList for the view.
+	 * @param genomes
+	 *          the list of all genomes
+	 */
+	private void createList(List<Genome> genomes) {
+		list = new JList(genomes
```
Comment (Delfil) [inline src/main/java/tudelft/ti2806/pl3/sidebar/genomeFilter/GenomeFilterView.java]: Raw type

File: src/main/java/tudelft/ti2806/pl3/sidebar/genomeFilter/GenomeFilterView.java [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+		}
+		list.setSelectedIndices(indices);
+	}
+
+	/**
+	 * Get all selected items.
+	 * @return
+	 *          the selected items in the list
+	 */
+	public List getSelected() {
```
Comment (Delfil) [inline src/main/java/tudelft/ti2806/pl3/sidebar/genomeFilter/GenomeFilterView.java]: Raw type

**Comment by mathieupost** [comment]
Helemaal mooie ding

**Pull request merged**